### ISSUE TITLE
Cannot return null for non-nullable field

### ISSUE THREAD

**Issue by b-water** [rebing/graphql-laravel#303]
After i ran "composer update" the following error occured:

[2019-06-03 12:03:18] local.ERROR: Cannot return null for non-nullable field Account.id.
GraphQL (1:41)
query name{accounts(tax_id_number: "9"){id company number created_at state { id code } role {id code}}}`

The Account.id Field declaration:
`            'id'                => [
                'type'        => Type::nonNull(Type::int()),
                'description' => 'The id of an account',
                'alias'       => 'account_id'
]`

I have only 2 rows in the account database table and both have an id (1, 2). The same Query worked before the composer update. Im using Lumen 5.8 and Rebing/GraphQL 1.22 on a PHP 7.2/MySQL 5.7.

**Comment by sicbb** [comment]
Same here. I started a new Lumen project and followed the installation and example instructions exactly.

On my GET request **/graphql?query=query+FetchUsers{users{id,email}}** i get the same error as mentioned above.

```
#41 {main} {"exception":"[object] (GraphQL\\Error\\InvariantViolation(code: 0): Cannot return null for non-nullable field User.id. at /Users/daniel/Documents/Web/Projects/test/lumen-graphql/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php:808)
[stacktrace]
```

However, if I delete all users, the error will no longer occur.

**Comment by mfn** [comment]
@sicbb are you too using an `alias`?

I suspect it has to do with this change https://github.com/rebing/graphql-laravel/pull/283 maybe?

**Comment by sicbb** [comment]
@mfn yes, i'm using **alias**

I removed the **alias** tag and the error no longer occur.

**Comment by b-water** [comment]
@mfn same with me, after removing the alias it works again

**Comment by mfn** [comment]
Was the `alias` there by accident / a leftover? I.e. does/did it serve any real purpose?

Previously the `alias` field on its own didn't do anything unless a custom resolver was present.

Which felt like a bug/oversight and reduced the intent/usefulness of the alias.

The new behaviour is now to implement a default resolver for such a field _to return the actual aliased_ column.

I assume in your case wasn't present or something.

TL;DR: is your code fixed by removing it or do you _need_ the alias that way?

**Comment by sicbb** [comment]
I followed the installation guide on:

> https://github.com/rebing/graphql-laravel#creating-a-query

At this point in the documentation, the `alias` is used. Didn't know the real purpose.

**Comment by b-water** [comment]
Same with me

**Comment by mfn** [comment]
The documentation was just a (not very practical, I guess) example.

Looks to me this issue is solved and was "configuration error" which is exposed in the latest release because of a recent bugfix in the library.

@b-water if you agree, please close the issue
